bpf/bpf_lxc.c

```diff
@@ -861,6 +861,17 @@ static inline int __inline__ ipv6_policy(struct __sk_buff *skb, int ifindex, __u
 	return 0;
 }
 
+/* Check whether the packet should be allowed despite unhandled IP fragments.
+ * Returns true if the packet should pass, false if it should be dropped. */
+static bool check_ip4_fragments(struct iphdr *ip4)
```

Comment:
We would need a similar check in the IPv6 case as well to make sure no fragments pass through here, no_

Reply:
Oh, I meant to file an issue for it, this is already the case, but it's handled in a slightly weird way, grep for `DROP_FRAG_NOSUPPORT`.

Reply:
See #3775.